resources/lang/en/events/lead_ceremony_df.json

```
@@ -128,7 +128,572 @@
+      "stubbornness", "authority"]
+        }
+      ]
+    },
+  
+    "strange_adult": {
+      "tags": [],
+      "lead_trait": [],
+      "star_trait": ["strange"],
+      "rank": ["warrior", "elder", "medicine cat", "mediator", "apprentice", "deputy", "medicine cat apprentice", "mediator apprentice"],
+      "life_giving": [
+        {
+          "text": "A shape begins to form in the distance, then in the space of a pawstep, r_c is suddenly directly in front of m_c's muzzle. Before m_c can flinch away, the unearthly spirit gives a life for [virtue], then vanishes into the darkness like smoke gusted by the breeze. Only a trace of {PRONOUN/r_c/poss} glowing eyes remains."
+        }
+      ]
+    },
+  
+    "strict_adult": {
+      "tags": [],
+      "lead_trait": [],
+      "star_trait": ["strict"],
+      "rank": ["warrior", "elder", "medicine cat", "mediator", "apprentice", "deputy", "medicine cat apprentice", "mediator apprentice"],
+      "life_giving": [
+      {
+        "text": "r_c looms over m_c, a fierce light kindled in the depths of {PRONOUN/r_c/poss} gaze. {PRONOUN/r_c/subject/CAP} {VERB/r_c/snarl/snarls} that m_c, as leader, is now responsible for the code itself. This duty must not be shirked. r_c gives a life for [virtue].",
+        "virtue": ["ambition", "bravery", "certainty", "clear judgment", "confidence", "courage",
+    "determination", "endurance", "farsightedness", "instincts", "strength", "ferocity", "cunning", "shamelessness", "zealotry", "bloodlust", "restless energy", 
+    "stubbornness", "authority"]
+      }
+      ]
+    },
+  
+    "thoughtful_adult": {
+      "tags": [],
+      "lead_trait": [],
+      "star_trait": ["thoughtful"],
+      "rank": ["warrior", "elder", "medicine cat", "mediator", "apprentice", "deputy", "medicine cat apprentice", "mediator apprentice"],
+      "life_giving": [
+        {
+          "text": "r_c watches m_c with piercing eyes, as though the dark spirit can see right through all of m_c's thoughts! {PRONOUN/r_c/subject/CAP} {VERB/r_c/are/is} silent as {PRONOUN/r_c/subject} {VERB/r_c/give/gives} a life for [virtue].",
+          "virtue": ["ambition", "bravery", "certainty", "clear judgment", "confidence", "courage",
+      "determination", "endurance", "farsightedness", "instincts", "strength", "ferocity", "cunning", "shamelessness", "zealotry", "bloodlust", "restless energy", 
+      "stubbornness", "authority"]
+        }
+      ]
+    },
+
+  	"troublesome_adult": {
+      "tags": [],
+      "lead_trait": [],
+      "star_trait": ["troublesome"],
+      "rank": ["warrior", "elder", "medicine cat", "mediator", "apprentice", "deputy", "medicine cat apprentice", "mediator apprentice"],
+      "life_giving": [
+        {
+          "text": "Whispering of a prank to pull on the other Clans, and how it would be SO funny, r_c gives m_c a life for [virtue].",
```

```suggestion
          "text": "r_c gives m_c a life for [virtue], simultaneously whispering a prank into {PRONOUN/m_c/poss} ear that would be <i>so funny</i> to pull on the other Clans.",
```
I don't think you wrote this because you mentioned it as a bugfix, but I'm also gonna rephrase this one too. #cantstopwontstop